MonoMod.RuntimeDetour/NativeDetour.cs

```diff
@@ -27,7 +27,8 @@ public unsafe class NativeDetour : IDetour {
         public bool IsValid { get; private set; }
         public bool IsApplied { get; private set; }
 
-        public readonly NativeDetourData Data;
+        private NativeDetourData _Data;
+        public NativeDetourData Data => _Data;
```

Comment:
This is technically a breaking change, but realistically almost nobody should be using this directly, and whoever is using this will probably recompile with an update anyway, bypassing the break. Just writing this down here for future reference.